test-series/1/test-1.2.0.ipynb

```diff
@@ -0,0 +1,1277 @@
+{
```

Comment:
<p>If it hasn't already been mentioned, should probably say how long a single file is.</p><p> </p><p>(I see it's mentioned below that it's 66 integrations. Maybe just mention here briefly that that's about 10(?) minutes.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Alrighty, I've done that. Thanks for the review!</p>